GSoC 2020: Add a OpenAPI link to the Open API specification project idea (#3161)
jenkins-infra/jenkins.io: content/projects/gsoc/2020/project-ideas/automatic-spec-generator-for-jenkins-rest-api.adoc

=== Background
Developers need to know what to expect in terms of responses for each REST API endpoint, so external tools like jenkins-rest can be developed with confidence, and possibly with the help of some REST specification automation.
For example, this could be done using the link:https://www.openapis.org/[OpenAPI Specification] (formerly known as Swagger Specification).

> The two main contents users of the REST API are looking for are:

Reviewer: Optional alternate phrasing
```suggestion
Users of the REST API most often want:
```
